### ISSUE TITLE
МКС-Робин-Нано-V3.1 and a new loader for rrf

### ISSUE THREAD

**Issue by Osettt** [makerbase-mks/MKS-Robin-Nano-V3.X#102]
Hello. At the moment there is a problem with the bootloader for the new board version v3.1. People who wanted to put RRF on their board flashed the bootloader according to the instructions. After flashing, the board goes into a permanent restart. In this regard, please upload the standard version of the bootloader and specify in the instructions that the bootloader firmware is only suitable for v3.0, not v3.x.

**Comment by MKS-hosiey** [comment]
Yes, the V3.1 motherboard does not have enough memory, so there is a problem with updating the RRF firmware

**Comment by Osettt** [comment]
Will you post the standard bootloader for those who reflashed the bootloader?

**Comment by Alex77-maker** [comment]
Hello. I have the same problem. Please send the standard bootloader .

**Comment by fahnrich7** [comment]
Привет. У меня та же проблема. Пожалуйста, пришлите стандартный загрузчик!!! Пожалуйста помогите!

**Comment by SnakeKVC** [comment]
Hello. I have the same problem. Please send a standard bootloader!!! Please, help!

**Comment by fahnrich7** [comment]
There is no bootloader! I bought a new board!
--
Отправлено из Mail.ru для Android среда, 24 января 2024г., 16:06 +06:00 от SnakeKVC  ***@***.*** :
